holidayextras/ui-brand-hx
# Restyled block quotes and inline quotes

#### What does this PR do? (please provide any background)
Restyle block quotes and inline quotes

#### What tests does this PR have?
N/A

#### How can this be tested?
You can test this by using https://github.com/holidayextras/pattern-library
In the package.json switch the UI-BRAND-HX call to use this branch. If you don't know how, then please ask.

Check out these changes on...
pattern-library/atoms/articles

#### Screenshots / Screencast

Before
![blockquotes before](https://cloud.githubusercontent.com/assets/400087/12269881/6c27ccea-b94b-11e5-9ec8-b5795aed9eab.png)

After
![blockquotes after](https://cloud.githubusercontent.com/assets/400087/12269886/6ef12e12-b94b-11e5-8d9c-844bd586a0c9.png)


#### What gif best describes how you feel about this work?
![](http://i.giphy.com/3o85xHKDE8qUL0CfZe.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
